library/unwind/build.rs

```diff
@@ -4,6 +4,11 @@ fn main() {
     println!("cargo:rerun-if-changed=build.rs");
     let target = env::var("TARGET").expect("TARGET was not set");
 
+    if cfg!(feature = "system-llvm-libunwind") {
+        println!("cargo:rustc-link-lib=unwind");
```

Comment:
I think this is specified already in https://github.com/rust-lang/rust/blob/3d8608a8638fabc46e83acc0626d4cf2782f0702/library/unwind/src/lib.rs#L62 - can we early return here?

Reply:
fixed